README.md

```diff
@@ -506,6 +506,8 @@ In case of using 'ssh', one may need additional steps to ensure that the bot use
   ssh-add ~/.ssh/NAME_OF_PRIVATE_KEY_FILE
   ```
 
+Note that the `bot: status` command doesn't work with SSH keys; you'll still need a Github token for that to work.
```

Comment:
I'm confused...

The `bot: status` command doesn't involve any cloning of GitHub repositories, so what does it do that doesn't work with SSH keys?

Reply:
it uses the Github API to fetch all the comments in the PR to generate a status report